### PR TITLE
Made responses in Getting Started code into valid Proxy Integration r…

### PR DESCRIPTION
**Pull request by nikioftime**
…esponses

Running through the HOWTO.md as stated, I got `ERROR: Function Products returned an invalid response (must include one of: body, headers or statusCode in the response object)` as a result of running `curl http://localhost:3000/products`, because the responses in the callback aren't valid Proxy Integration responses. This change simply puts those original responses into the body of a JSON object.

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -58,23 +58,23 @@ exports.handler = (event, context, callback) => {
         case "GET":
             
             if(id) {
-                callback(null, "This is a READ operation on product ID " + id);
+                callback(null, { body: "This is a READ operation on product ID " + id});
```
Comment (sanathkr) [inline HOWTO.md]: Awesome! It is generally a good practice to manually return statusCode = 200 for successful calls. Can you make that change as well?

**Comment by sanathkr** [comment]
Thanks for doing this!

File: HOWTO.md (hunk 1 above)
Comment (nikioftime) [inline HOWTO.md]: Of course!

**Comment by nikioftime** [comment]
Glad I could help :)

**Pull request merged**